fe/fe-core/src/main/java/com/starrocks/catalog/IcebergTable.java

```diff
@@ -92,7 +92,8 @@ public class IcebergTable extends Table {
     private org.apache.iceberg.Table nativeTable; // actual iceberg table
     private List<Column> partitionColumns;
     // used for recording the last snapshot time when refresh mv based on mv.
-    private long refreshSnapshotTime = -1L;
+    @SerializedName(value = "refreshSnapshotTime")
+    private AtomicLong refreshSnapshotTime = new AtomicLong(-1);
```

Comment:
It will be updated after mv refresh success and may be queried at concurrency.
```
            if (baseTable.isIcebergTable()) {
                IcebergTable icebergTable = (IcebergTable) baseTable;
                icebergTable.setRefreshSnapshotTime(snapshotInfo.getRefreshSnapshotTime());
                // iceberg table use ALL as partition name, so we don't need to remove partition info
                currentTablePartitionInfo.keySet().removeIf(partitionName ->
                        (!partitionNames.contains(partitionName) && !partitionName.equals(ICEBERG_ALL_PARTITION)));
            } else {
                currentTablePartitionInfo.keySet().removeIf(partitionName -> !partitionNames.contains(partitionName));
            }
```

Reply:
why refreshSnapshotTime need to be update after mv refresh success